.github/scripts/ensure_actions_will_cancel.py

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env python3
 
 import sys
-
-from pathlib import Path
-
 import yaml
+from pathlib import Path
```

Comment:
How this change is related to the PR title?

Reply:
Since `usort` recognizes `yaml` as stdlib, it will group all first-party imports.

Before: `usort` does not recognize `yaml`, it will be treat as third-party by default. (FYI, `isort` knows `yaml` is stdlib)

```python
# first-party
import sys

from pathlib import Path

# thrid-party
import yaml
```

After:

```python
# first-party
import sys
import yaml
from pathlib import Path
```